services/explorer/src/utils/networks.ts

```diff
@@ -0,0 +1,57 @@
+import { SELECTED_NETWORK, Network } from './Enums';
+import {
+  GRAPHQL_API_TESTNET,
+  GRAPHQL_API_DEVNET,
+  GRAPHQL_API_MAINNET,
+  MAINNET_API,
+  TESTNET_API,
+  DEVNET_API,
+} from '../api';
+
+export const recreateGraphQLConnection = (): string => {
+  const network = localStorage.getItem(SELECTED_NETWORK);
```

Comment:
We should not take the network directly from localStorage. There is a a hook useNetworkSelection which sets inside the context the current network (and it has a logic on deciding if it takes it first based on the curent domain or local storage and so on). So this method should receive the network as parameter.